src/main/java/io/appform/dropwizard/sharding/config/ShardedHibernateFactory.java

```diff
@@ -43,4 +43,5 @@ public class ShardedHibernateFactory {
 
     @Valid
     private BlacklistConfig blacklist;
+
```

Comment:
Why not add `CustomDatabaseConfig` here with an member called `options`? This will make it configurable at startup instead of needing code changes

Reply:
added shardingOptions field in this bean. kept default as false